markdown/the_photomode_wishlist.md

```diff
@@ -78,7 +78,9 @@ Letting the player disable depth of field (that being from a cutscene or gamepla
 
 Letting the player spawn their own lights to re-illuminate the characters and scene can be a game changer. The controls and UI can be tricky to get right, but we suggest looking at what Insomniac has done with the Spider-Man games and Ratchet & Clank: Rift Apart. And, of course, what Frans has been doing with the [UUU](https://opm.fransbouma.com/uuuv5.htm#lights).
 
-Besides letting the player spawn lights, it's almost equally important to let them turn off the in-game lights. That way they can start lighting the scene from scratch without having to deal with the other lights that don't benefit their composition. An example of what being able to turn off the in-game lights and relit a scene by yourself can achieve:
+Besides letting the player spawn lights, it's almost equally important to let them turn off the in-game lights. That way they can start lighting the scene from scratch without having to deal with the other lights that don't benefit their composition.
+
+Of all of the features listed in this article, this one might be one of the most expensive ones to make. But, as the example below shows, being able to turn off the in-game lights and relit a scene by yourself might be one of the most transformative tools screenshoters have.
```

Comment:
```suggestion
Of all of the features listed in this article, this one likely requires more effort to implement. But, as the example below shows, being able to turn off the in-game lights and re-light a scene completely is one of the most transformative tools screenshotters have.
```

A couple of typos, and reworded a bit to be a bit more "direct".

Reply:
Lol sorry, I have been so exposed to "lit" tools that the correct wording has totally been forgotten 😆